mfhdf/dumper/hdp.h

```diff
@@ -17,7 +17,7 @@
 #include "hdf.h"
 #include "hfile.h"
 
-/* Global Variables (ick) */
+/* Global Variables */
```

Comment:
We should be able to eliminate this global vinit_done as it only exists in two files; hdp_list.c and hdp_util.c

Reply:
I think the usage in hdp_util.c could be a simple function call.